clang/test/APINotes/Inputs/Headers/SwiftImportAs.apinotes

```diff
@@ -3,6 +3,12 @@ Name: SwiftImportAs
 Tags:
 - Name: ImmortalRefType
   SwiftImportAs: reference
+  Methods:
+    - Name: methodReturningFrt__
+    - Name: methodReturningFrt_returns_unretained
+      SwiftReturnOwnership: 'returns_unretained'
+    - Name: methodReturningFrt_returns_retained
+      SwiftReturnOwnership: 'returns_retained'
```

Comment:
Let's drop `returns_` prefix from the attribute value to make it more concise. The word `Return` is already in the attribute name, so we don't get any extra information by repeating it in the attribute value.